components/patient-data/impl/src/main/java/org/phenotips/data/internal/PhenoTipsPatient.java

```diff
@@ -139,7 +139,6 @@ public PhenoTipsPatient(XWikiDocument doc)
         this.disorders = Collections.unmodifiableSet(this.disorders);
 
         loadSerializers();
```

Comment:
Can we take this one step further and only load the serializers on the first call to `getData`? :)